src/test/java/com/example/programmarsjpa/domain/CustomerTest.java

```diff
@@ -0,0 +1,93 @@
+package com.example.programmarsjpa.domain;
+
+
+import com.example.programmarsjpa.domain.customer.Customer;
+import com.example.programmarsjpa.domain.customer.CustomerRepository;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.*;
+
+@Transactional
+@SpringBootTest
```

Comment:
```suggestion
@DataJpaTest
```

Repositoty 영역의 테스트이다보니 굳이 SpringBootTest를 사용하지 않아도 되보입니다. 
@DataJpaTest 내에 @Transactional 도 있어서 하나로 사용 가능해 보이는데 어떻게 생각하시나요?